gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/flow/mediators/builtin/flowcontrollers/filter/Source.java

```diff
@@ -29,11 +31,24 @@
 
     private String value;
 
+    private PATHLANGUAGE pathLanguage;
```

Comment:
any reason as to why it has to be all caps ? whats the best practice around capitalization (for class/enum names) ?